proto/src/determined/experiment/v1/searcher.proto

```diff
@@ -165,12 +165,12 @@ message ShutdownOperation {
   int32 placeholder = 1;
 }
 
-// SearcherOperation is an operation issued by the searcher.
+// SearcherOperation is an operation issued by the custom searcher.
 message SearcherOperation {
   // A searcher operation is one of the following operations.
   oneof union {
-    // ValidateAfter is issued to tell a trial to train some then validate.
-    ValidateAfterOperation validate_after = 1;
+    // TrialOperation is issued for the current trial. .
+    TrialOperation trial_op = 1;
```

Comment:
I'd prefer to have the name `trial_operation` so we don't have this mismatch between the type name and the field name.

Reply:
yup sorry missed this.